messaging/Controllers/BundlesController.cs

```diff
@@ -77,15 +79,29 @@ public async Task<ActionResult<Bundle>> GetOutgoingMessageItems(string jurisdict
                 return BadRequest("Pagination does not support specifying a page without a _since parameter");

```

Comment:
I tried requesting the second page with the businessIDs and _count=1 and got this error back 
"Pagination does not support specifying a page without a _since parameter"
The if statement on line 76 needs to be updated to also check if certificateNumber == null AND deathYear == null